technical-writing-project-ideas.md

```diff
@@ -1,58 +0,0 @@
----
-title: TiDB Technical Writing Project Ideas
```

Comment:
@lilin90 PTAL, thanks~

Reply:
@TomShawn We can remove this file. It's outdated and not necessary to keep it in this repo now.

Reply:
Get. Thanks!